Tasks/JenkinsDownloadArtifacts/ArtifactDetails/JenkinsRestClient.ts

```diff
@@ -176,59 +183,95 @@ export class JenkinsRestClient {
         });
     }
 
+    private GetClient(): HttpClient {
+        tl.debug(`Creating http client`);
+
+        const endpoint = tl.getInput("serverEndpoint", true);
+        const strictSSL: boolean = ('true' !== tl.getEndpointDataParameter(endpoint, 'acceptUntrustedCerts', true));
+
+        let requestOptions: IHttpRequestOptions = {
+            ignoreSslError: !strictSSL
+        };
+
+        let proxyUrl: string = tl.getVariable("agent.proxyurl");
```

Comment:
https://github.com/Microsoft/vsts-task-lib/blob/master/node/mock-task.ts does not have mock implementation for getHttpProxyConfiguration. Hence checking for variable and and calling to avoid mock failure in test.